content/webapp/services/prismic/transformers/exhibition-guides.ts

```diff
@@ -77,38 +87,35 @@ export function transformExhibitionGuideToExhibitionGuideBasic(
   }))(exhibitionGuide);
 }
 
-function transformRelatedExhibition(exhibition): Exhibit {
-  return {
-    exhibitType: 'exhibitions',
-    item: undefined,
-    id: exhibition.id,
-    title: (exhibition.data && asText(exhibition.data.title)) || '',
-    description:
-      exhibition.data &&
-      asText(exhibition.data.promo[0].primary.caption[0].text),
-  };
+function transformRelatedExhibition(
+  exhibition: prismicT.RelationField<
+    'exhibitions',
+    'en-gb',
+    InferDataInterface<ExhibitionPrismicDocument>
+  >
+): Exhibit | undefined {
+  return isFilledLinkToDocumentWithData(exhibition)
+    ? {
+        exhibitType: 'exhibitions',
+        item: undefined,
+        id: exhibition.id,
+        title: asTitle(exhibition.data.title),
+        description:
+          (exhibition.data?.promo[0] &&
+            asText(exhibition.data.promo[0]?.primary.caption)) ||
+          '',
+      }
+    : undefined;
 }
 
-function transformYoutubeEmbed(embed) {
```

Comment:
I wanted to type this, and in the process I ended up moving it out to a standalone file.